### PR TITLE
Add an alias for the pool

### PR DESCRIPTION
**Pull request by jo66**
## Subject

I am targeting this branch, because it's feature.

## Changelog

```markdown
### Added
- Added an alias for the pool
```

### CODE REVIEW

**Comment by jo66** [comment]
@greg0ire Is it so simple?

File: tests/Provider/PoolTest.php [hunk 1]
```diff
@@ -73,6 +76,16 @@ public function testGetInvalidProviderName()
...
 
+    public function testPoolAlias()
+    {
+        $container = new ContainerBuilder();
+        $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../../src/Resources/config'));
+        $loader->load('provider.xml');
+        $pool = $container->get('sonata.media.pool');
+        $alias = $container->get('%sonata.media.pool.class%');
+        $this->assertEquals($pool, $alias);
+    }
```
Comment (greg0ire) [inline tests/Provider/PoolTest.php]: Sorry I was unclear (I was on my phone at that time), I was thinking about this test: https://github.com/sonata-project/SonataMediaBundle/blob/3.x/tests/DependencyInjection/SonataMediaExtensionTest.php

Comment (jo66) [reply tests/Provider/PoolTest.php]: Done.

**Comment by greg0ire** [comment]
It seems you are having troubles with git. Next time you amend a commit, use `git push --force`, it will tell git to overwrite whatever it is you have pushed before, and not make a merge commit like it did. Also, please kindly squash your commits together. If you don't, we'll try to remember to do it for you but it's best if you save us this trouble.

How to do that?

1. `git rebase -i origin/3.x`, assuming `origin` is a git remote that points to this repository, and not your fork. If you're not sure what your remotes are, run `git remote -vvv`, there should be your fork and the holy/reference/base/origin/whatever-you-call-it repository.
2. A window will show up with many lines, replace `pick` with `fixup` on every line but the first one
3. Close your editor, git should do its magic, and you should end up with one commit
4. Use `git push --force` to overwrite what you already pushed. Don't forget the `--force` option otherwise git will try to merge both things together.

**Comment by jo66** [comment]
@greg0ire 
> Next time you amend a commit, use `git push --force`, it will tell git to overwrite whatever it is you have pushed before, and not make a merge commit like it did

Thanks for advise.

Commits squashed.

**Comment by greg0ire** [comment]
Thanks @jo66 !

**Pull request merged**